tests/test_package_python_function.py

```diff
@@ -34,4 +37,19 @@ def test_package_python_function(tmp_path: Path) -> None:
     ]
     main()
 
-    assert (output_dir_path / 'project_1.zip').exists()
\ No newline at end of file
+    zip_file = output_dir_path / "project_1.zip"
+    assert zip_file.exists()
+
+    verify_dir = tmp_path / "verify"
+    verify_dir.mkdir()
+    with zipfile.ZipFile(zip_file, "r") as zip:
+        zip.extractall(verify_dir)
+        for file_info in zip.infolist():
+            mode = (file_info.external_attr >> 16) & 0xFFFF
+            assert mode == EXPECTED_FILE_MODE
+            assert file_info.date_time == EXPECTED_FILE_DATE_TIME
+
+        assert verify_dir / "project_1" / "__init__.py"
```

Comment:
These asserts don't appear to actually checking anything.  Perhaps you intended to call .exists() ?

Reply:
ack!!! no wonder it always passed 🤦🏻 yea I'll fix that